### ISSUE TITLE
qt 5.12.4 minimal install is borken

### ISSUE THREAD

**Issue by AliAskari** [bincrafters/community#1013]
### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **qt/5.12.4**
  * Operating System+version: **Linux Ubuntu 18.04**
  * Compiler+version: **gcc-7.4, aarch64-linux-gnu-gcc, 7.4**
  * Conan version: **conan 1.19.2**
  * Python version: **Python 3.6.8**

### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
Configuration for profile aarch64-linux:

[settings]
os_build=Linux
arch_build=x86_64
os=Linux
arch=armv8
compiler=gcc
compiler.version=7
compiler.libcxx=libstdc++
build_type=Debug
[options]
[build_requires]
[env]
CONAN_CMAKE_TOOLCHAIN_FILE=/tmp/aarch64-linux-toolchain.cmake
CONAN_CMAKE_FIND_ROOT_PATH=/usr/aarch64-linux-gnu
CHOST=aarch64-linux-gnu
AR=/usr/bin/aarch64-linux-gnu-ar
ASM=/usr/bin/aarch64-linux-gnu-as
AS=/usr/bin/aarch64-linux-gnu-as
RANLIB=/usr/bin/aarch64-linux-gnu-ranlib
CC=/usr/bin/aarch64-linux-gnu-gcc
CXX=/usr/bin/aarch64-linux-gnu-g++
STRIP=/usr/bin/aarch64-linux-gnu-strip
RC=/usr/bin/aarch64-linux-gnu-windres
```

### Steps to reproduce (Include if Applicable)

I'm trying to install the most minimal qt 5.12.4 for both linux  and linux armv8. Both of them are failing with the exact same traces.
This is the conan recipe I am using:
```
[requires]
zlib/1.2.11@conan/stable
bzip2/1.0.8@conan/stable
qt/5.12.4@bincrafters/stable

[generators]
cmake

[options]
qt:opengl=no
qt:openssl=False
qt:GUI=False
qt:widgets=False
qt:with_pcre2=False
qt:with_glib=False
qt:with_doubleconversion=False
qt:with_freetype=False
qt:with_harfbuzz=False
qt:with_libjpeg=False
qt:with_libpng=False
qt:with_sqlite3=False
qt:with_mysql=False
qt:with_pq=False
qt:with_odbc=False
qt:with_sdl2=False
qt:with_libalsa=False
qt:with_openal=False
```

Basically all options that are `True` by default are set to `False`. 

### Logs (Include/Attach if Applicable)
I have attached the log files from both native and cross builds.
[conan_native_log.txt](https://github.com/bincrafters/community/files/3770135/conan_native_log.txt)
[conan_cross_log.txt](https://github.com/bincrafters/community/files/3770136/conan_cross_log.txt)

**Comment by ericLemanissier** [comment]
Thanks for the precise report. The actual error is the following:
```
ERROR: Your C library does not provide sscanf_l or snprintf_l.
You need to use libdouble-conversion for double/string conversion.
```
sscanf_l and snprintf_l are considered missing because of this error `fatal error: xlocale.h: No such file or directory`. Can you try to install libc++-dev package matching your target architecture ?
Alternatively, you can try do use option qt:with_doubleconversion=True

**Comment by AliAskari** [comment]
Yeah, that was it. Thanks!
